### PR TITLE
Explain how to force use of IPv4 to download packages, refs #1

### PR DESCRIPTION
**Pull request by clochix**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,3 +79,12 @@ from the host. So far, so good. The tricky part is being able to run apps on
 the host while they appear from within the docker. The hack used here is to
 use SSH port tunnelling between the container and the host. That's what the JS
 tool `index` is used for.
+
+
+## Troubleshooting
+
+**`sync.sh` hangs when trying to download packages.**
```
Comment (bnjbvr) [inline README.md]: I think there's no such thing as `sync.sh`?

**Comment by bnjbvr** [comment]
Thank you for the PR! Just a quick tweak is needed in the readme before i can merge.

File: README.md (hunk 1 above)
Comment (clochix) [inline README.md]: Sorry, it's `build.sh`. I should'nt try do do more than one thing at the same time. I should drink more coffee.

**Comment by bnjbvr** [comment]
Thank you for the PR!

**Pull request merged**